composer.json

```
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^7.2 || ^8.0",
-        "doctrine/orm": "^2.15"
+        "doctrine/orm": "^2.19 || ^3.0"
```

```suggestion
        "doctrine/orm": "^2.19 || ^3.0 @dev"
```

This should fix the CI on PHP 7 for the time being. We can revert it once ORM 2.19 has been released.